src/core/turn-order.js

```diff
@@ -175,6 +164,27 @@ export function UpdateActivePlayersOnceEmpty(ctx) {
   };
 }
 
+/**
+ * Apply an active player argument to the given player ID
+ * @param {Object} ctx A context object
+ * @param {String} playerID The player to apply the parameter to
+ * @param {(String|Object)} arg An active player argument
+ */
+function ApplyActivePlayerArgument(ctx, playerID, arg) {
+  if (typeof arg !== 'object' || arg === null) {
```

Comment:
We should use `Stage.NULL` instead of `null` directly here.

Reply:
Are you sure? This is checking we have a valid object (`typeof null === 'object'`, but it’s a `TypeError` to try to read its properties) rather than whether the argument is the null stage. If in the future `Stage.NULL` got switched to `'null-stage'` or something, that would produce errors here.

Reply:
I'm probably missing something here.  Is this code not trying to (effectively) say:

```js
if (argument is a stage name or the null stage) {
  arg = { stage: arg };
}

// from here arg is guaranteed to be an object (either passed in directly by the user or set by us above).
```

Is there any other possible reason someone might pass `null` to `arg` if not to indicate the null stage?

Reply:
I was more going for `if (argument is really an object)`. You’re right that for now, `null` would usually indicate `Stage.NULL`.

A user might pass in `null` directly or have a buggy function that returns `null` unintentionally. I guess that if that happens, that’s a user error in any case, so I can change it to `Stage.NULL` if you prefer.

Reply:
No, it's fine as is.  Just wanted to clarify.

Reply:
Done in 5c357a3.

(Sorry to be picky — just that until there’s some more checking around whether a stage argument is valid, it feels a bit weird to test against `Stage.NULL`)